### PR TITLE
Combined 10 Dependabot Pull Requests

### PR DESCRIPTION
**Pull request by miketheman**
* Closes #11922
* Closes #11924
* Closes #11925
* Closes #11941
* Closes #11942
* Closes #11943
* Closes #11944
* Closes #11945
* Closes #11946
* Closes #11947
* Closes #11948

### CODE REVIEW

**Comment by miketheman** [comment]
Once merged, should unblock #11917

File: requirements/lint.txt
Comment (di) [inline requirements/lint.txt:317]: Any idea why this is being removed?

Comment (miketheman) [reply requirements/lint.txt:317]: Yes, types-s3transfer dropped it in https://github.com/youtype/types-s3transfer/commit/16b7b3a07e76aa4acfdb035967143f03293c272f

**Review by di — APPROVED** [review]

**Pull request merged**
